apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -90,8 +90,9 @@ const chooseSharingNameModal = (resLocals) => {
           <p class=form-text>
             Enter the sharing name you would like for your course.
           </p>
-          <p><strong>Once you choose your course's sharing name, you will not be able to change it</strong>,
+          <p><strong>Once you choose your course's sharing name, you may not be able to change it</strong>,
             because doing so would break the assessments of other courses that have imported your questions.
+            <strong>If you have not yet shared any questions publicly or with other courses, you may edit your course's sharing name.</strong>
```

Comment:
We should be able to figure out whether or not renaming is allowed at the instant the page is loaded and update the UI to say concretely "you can/cannot rename this" and disable buttons/etc. appropriately.

Reply:
@Michael-Holmes-SDE we talked about this in our developer meeting and decided that the easiest thing to do is this: The 'Choose sharing name' button on the page should always be enabled. 

If you click on it and you are allowed to change it, the dialog you see should say that you will be allowed to edit it until you share questions.

If you click on it and you are not allowed to change it, the dialog should say that you cannot change the sharing name, because you already have shared questions--and then the dialog should not have the 'choose sharing name' button on it, just the close button to close the dialog.